specs/bond-manager.md

```diff
@@ -0,0 +1,100 @@
+# Bond Manager Interface
+
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+**Table of Contents**  *generated with [DocToc](https://github.com/thlorenz/doctoc)*
+
+- [Overview](#overview)
+- [The Bond Problem](#the-bond-problem)
+  - [Simple Bond](#simple-bond)
+  - [Variable Bond](#variable-bond)
+- [Types of Bond Managers](#types-of-bond-managers)
+  - [OracleBondManager](#oraclebondmanager)
+  - [AttestationBondManager](#attestationbondmanager)
+  - [FaultBondManager](#faultbondmanager)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+## Overview
```

Comment:
Could we talk about the bond manager in a more generic sense first, without connection to the `L2OutputOracle`? Starting off with what a bond is (value) that acts as economic security and it can be claimed if there is proof of malicious intent? Something like that